.github/workflows/dapr.yml

```diff
@@ -204,6 +202,11 @@ jobs:
         with:
           name: dapr_darwin_amd64
           path: ${{ env.ARTIFACT_DIR }}
+      - name: download artifacts - dapr_darwin_arm64
+        uses: actions/download-artifact@master
+        with:
+          name: dapr_darwin_arm64
+          path: ${{ env.ARTIFACT_DIR }}
       - name: download artifacts - dapr_grafana_dashboards
         uses: actions/download-artifact@master
         with:
```

Comment:
You might need to add to the "tar release binaries" step too.

Reply:
Linux only